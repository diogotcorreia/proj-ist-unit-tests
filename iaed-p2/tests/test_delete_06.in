set abc 123
set b/////////////adfgdf/azdgd/dhfhj/sdghst 5
set aabc /b
set hsbdguj/dfhfhd rdfh
set /////////////vbdfuys/isdobgubf/sifbiweuf/abc 7
list
delete abc
print
list
quit
