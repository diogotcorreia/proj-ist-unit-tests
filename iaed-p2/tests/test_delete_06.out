aabc
abc
b
hsbdguj
vbdfuys
/b/adfgdf/azdgd/dhfhj/sdghst 5
/aabc /b
/hsbdguj/dfhfhd rdfh
/vbdfuys/isdobgubf/sifbiweuf/abc 7
aabc
b
hsbdguj
vbdfuys
